src/ImageSharp/Processing/Processors/Normalization/AdaptiveHistEqualizationSWProcessor.cs

```diff
@@ -51,8 +51,9 @@ protected override void OnFrameApply(ImageFrame<TPixel> source, Rectangle source
             Span<TPixel> pixels = source.GetPixelSpan();
 
             var parallelOptions = new ParallelOptions() { MaxDegreeOfParallelism = configuration.MaxDegreeOfParallelism };
-            int tileHeight = source.Height / this.Tiles;
-            int pixeInTile = tileHeight * tileHeight;
+            int tileWidth = source.Width / this.Tiles;
+            int tileHeight = tileWidth;
```

Comment:
Yes they are in this implementation. The OpenCV implementation provides the option to choose the number of tiles in X direction and in Y-Direction, but i cannot see a real usecase where this could be beneficial.